vtex/billing-app
# Make contracts endpoint return only active contracts

#### What is the purpose of this pull request?
Make contracts query return only active contracts

#### What problem is this solving?
The Billing `contracts/apps` endpoint returns **all** apps contracts of an account, regardless of being active or not. This query, meant to be used to get active contracts, filters the Billing contracts list by using the IsActive flag.

#### How should this be manually tested?
1. Clone/pull this repo/branch
2. Try this query in GraphiQL:
```
query {
  contracts {
    appId
    startDate
  }
}
```
This should return the active contracts in your account

#### Types of changes
